src/shared/components/mutationTable/column/DiscreteCNAColumnFormatter.tsx

```diff
@@ -18,15 +18,129 @@ export default class DiscreteCNAColumnFormatter {
         "-2": "deepdel"
     };
 
-    public static renderFunction(data: Mutation[], molecularProfileIdToMolecularProfile: {[molecularProfileId:string]:MolecularProfile}, cache:DiscreteCNACache) {
+    private static facetsCallTable:{[key:string]:facetsCall:string} = {
+        "no WGD00":"Homdel",
+        "no WGD10":"Hetloss",
+        "no WGD20":"CNLOH",
+        "no WGD30":"CNLOH & Gain",
+        "no WGD40":"CNLOH & Gain",
+        "no WGD50":"Amp (LOH)",
+        "no WGD60":"Amp (LOH)",
+        "no WGD11":"Diploid",
+        "no WGD21":"Gain",
+        "no WGD31":"Gain",
+        "no WGD41":"Amp",
+        "no WGD51":"Amp",
+        "no WGD61":"Amp",
+        "no WGD22":"Tetraploid",
+        "no WGD32":"Amp",
+        "no WGD42":"Amp",
+        "no WGD52":"Amp",
+        "no WGD62":"Amp",
+        "no WGD33":"Amp (Balanced)",
+        "no WGD43":"Amp",
+        "no WGD53":"Amp",
+        "no WGD63":"Amp",
+        "WGD00":"HOMDEL",
```

Comment:
@averyniceday this is still in all caps :)

Reply:
I also accidentally deleted half the mappings :| But it's back now! hahaha